src/main/java/com/springweb/dv_spring_web_mongo/logging/ProjectLoggingAspect.java

```diff
@@ -0,0 +1,56 @@
+package com.springweb.dv_spring_web_mongo.logging;
+
+import com.springweb.dv_spring_web_mongo.model.User;
+import com.springweb.dv_spring_web_mongo.service.ProjectService;
+import lombok.RequiredArgsConstructor;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Pointcut;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StopWatch;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Component
+@Aspect
+@RequiredArgsConstructor
+public class ProjectLoggingAspect {
```

Comment:
The aspect will wrap all controller methods, not only `ProjectController`. So, the class name should be more generic like this: `HttpRequestLoggingAspect`